summary-visualiser/templates/scenarios/write_get_agent_activity.html.tmpl

```diff
@@ -0,0 +1,33 @@
+{{ define "write_get_agent_activity_description" }}
+<p>A scenario where <b><code>write</code></b> peers creates entries, while <b><code>get_agent_activity</code></b> peers each query a single <code>write</code> agent's activity with <code>get_agent_activity</code>.
+
+Before a target <code>write</code> peer and the requesting <code>get_agent_activity</code> peer are in sync, this will measure the <code>get_agent_activity</code> call performance over a network. Once a <code>write</code> peer reaches sync with a <code>get_agent_activity</code> peer, the <code>write</code> peer will publish their actions and entries, and so the <code>get_agent_activity</code> calls will likely have most of the data they need locally. At that point this measures the database query performance and code paths through host functions.
+{{ end }}
+
+{{ template "scenario_summary" (dict
+    "title" "Write/<code>get_agent_activity</code>"
+    "description" (tmpl.Exec "write_get_agent_activity_description")
+    "data" .run_summary
+) }}
+
+<div class="scenario-metrics">
+    {{ with .scenario_metrics }}
+        {{ template "scenario_metric" (dict
+            "name" "Highest observed <code>action_seq</code>"
```

Comment:
I'm questioning whether this is a useful metric to even show?

Reply:
I don't think it's a problem to show it.